File: app/src/main/java/com/example/letmebeyourchef/Listeners/RicetteSimiliListener.java
package com.example.letmebeyourchef.Listeners;

import com.example.letmebeyourchef.Models.ResponseFromApiRicetteSimili;

import java.util.List;

public interface RicetteSimiliListener {
    void didFetch(List<ResponseFromApiRicetteSimili> response, String message);
    void didError(String message);
}

File: app/src/main/java/com/example/letmebeyourchef/listeners/RicetteSimiliListener.kt
package com.example.letmebeyourchef.listeners

import com.example.letmebeyourchef.recipeModels.ResponseFromApiRicetteSimili

open interface RicetteSimiliListener {
    fun didFetch(response: List<ResponseFromApiRicetteSimili>, message: String?)
    fun didError(message: String?)
}